system_baseline/openapi/api.spec.yaml

```diff
@@ -201,6 +201,96 @@ paths:
           $ref: '#/components/responses/Forbidden'
         '500':
           $ref: '#/components/responses/InternalServerError'
+  /baselines/{baseline_id}/systems:
```

Comment:
This is great for the portion needed for frontend for the new feature.  What about a way for hsps to perform the query of what baselines are associated with one given system id?  If that info is still stored only in the baselines db, then we still need an api endpoint for hsp to call and my understanding from beav is that it needs to be much higher, like at the same level as /mgmt/, we'll add an /internal/.  Should that be a separate PR?  If so, I'll try to work on that, just let me know.

Reply:
That will most probably will be different PR.

Also is this the way we want to go?
I thought that we will use that shared secret approach and the API endpoint will not be internal.

Reply:
I think it's sort of both to keep people from attempting to break it?  Either way, as you say, we'll discuss it on its own PR. :)